<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPrincipal</class>
 <widget class="QMainWindow" name="FormPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormPrincipal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>464</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,5,1,0">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>| Vendas abertas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>423</width>
           <height>191</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="layoutVendas"/>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
        <item>
         <widget class="QPushButton" name="novaVendaButton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Nova Venda</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
padding: 5px;
border: none;
border-radius: 7px;
background-color:  #1a1a8c;
color: #fff;
}

:hover {
background-color: #333399;
}</string>
          </property>
          <property name="text">
           <string>Nova Venda</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>277</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGest_o">
    <property name="title">
     <string>Gestão</string>
    </property>
    <addaction name="actionEstoque"/>
    <addaction name="actionFuncion_rios"/>
    <addaction name="actionVendas"/>
    <addaction name="actionFornecedores"/>
   </widget>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <widget class="QMenu" name="menuSobre">
     <property name="title">
      <string>Sobre</string>
     </property>
     <addaction name="actionSobre_o_Qt_Creator"/>
     <addaction name="actionStockBack"/>
    </widget>
    <addaction name="actionSair"/>
    <addaction name="menuSobre"/>
   </widget>
   <addaction name="menuGest_o"/>
   <addaction name="menuSistema"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
  </widget>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/login.png</normaloff>:/imgs/imagens/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionSobre_o_Qt_Creator">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/qtLogo.png</normaloff>:/imgs/imagens/qtLogo.png</iconset>
   </property>
   <property name="text">
    <string>Qt Creator</string>
   </property>
  </action>
  <action name="actionEstoque">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/estoque.png</normaloff>:/imgs/imagens/estoque.png</iconset>
   </property>
   <property name="text">
    <string>Estoque</string>
   </property>
  </action>
  <action name="actionFuncion_rios">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/funcionario.png</normaloff>:/imgs/imagens/funcionario.png</iconset>
   </property>
   <property name="text">
    <string>Funcionários</string>
   </property>
  </action>
  <action name="actionVendas">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/vendas.png</normaloff>:/imgs/imagens/vendas.png</iconset>
   </property>
   <property name="text">
    <string>Vendas</string>
   </property>
  </action>
  <action name="actionFornecedores">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/fornecedor.png</normaloff>:/imgs/imagens/fornecedor.png</iconset>
   </property>
   <property name="text">
    <string>Fornecedores</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/sair.png</normaloff>:/imgs/imagens/sair.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionStockBack">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/imgs/imagens/logo32PeB.png</normaloff>:/imgs/imagens/logo32PeB.png</iconset>
   </property>
   <property name="text">
    <string>StockBack</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
